lib/ProjectContent.ts

```diff
@@ -0,0 +1,44 @@
+import { OctokitAction } from "./OctokitAction";
+import { components } from "@octokit/openapi-types/types.d";
+import { IssueOrPR } from "./IssueOrPR";
+
+export class ProjectContent {
+
+    private readonly action: OctokitAction;
+    private readonly columns: number[] = [];
+
+    private constructor(action: OctokitAction, columns: number[]) {
+        this.action = action;
+        this.columns = columns;
+    }
+
+    public static async FromColumn(action: OctokitAction, column_id: number): Promise<ProjectContent> {
+        const { data: column } = await action.rest.projects.getColumn({ column_id });
+        const project_id = parseInt(column.project_url.split("/").pop());
```

Comment:
I wonder if pop will create a copy of the array (it does change the length) and how this affects performance. 

Probably it's not a problem for this script that runs once but would be nice to know.

Reply:
I don't even know how to measure that thing :) I'll leave it a mystery, as it doesn't matter here